test/StringField_test.js

```diff
@@ -1501,6 +1501,9 @@ describe("StringField", () => {
   describe("FileWidget", () => {
     const initialValue = "data:text/plain;name=file1.txt;base64,dGVzdDE=";
 
+    const nonUriEncodedValue = "fileáéí óú1.txt";
```

Comment:
Please move these two lines inside the "should encode file name with encodeURIComponent" test case.

Reply:
👍